debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -486,6 +499,29 @@ protected Object convertHStore(Column column, Field fieldDefn, Object data, HSto
             return convertHstoreToMap(column, fieldDefn, data);
         }
 
+    private Object convertLtree(Column column, Field fieldDefn, Object data) {
+        return convertValue(column, fieldDefn, data, "", r -> {
+            if (data instanceof String) {
+                r.deliver(data);
+            }
+            else if (data instanceof PGobject) {
+                r.deliver(data.toString());
+            }
+        });
+    }
+
+    private Object convertLtreeArray(Column column, Field fieldDefn, Object data) {
```

Comment:
From the failing test results, I can see that I didn't handle the array correctly. But I don't know what should be the right way...

Reply:
What test failure do you get exactly?